### PR TITLE
Updated ManagedServices for GA (api-version 2019-06-01)

### PR DESCRIPTION
**Pull request by rajivnandivada**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
     No changes to cmdlets. Confirmed with Mark that a review is not required.
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)
     No changes to cmdlets.

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/ManagedServices/ManagedServices/ManagedServices.csproj [hunk 1]
```diff
@@ -1,14 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <PsModuleName>ManagedServices</PsModuleName>
+    <AssemblyName>s</AssemblyName>
```
Comment (markcowl) [inline src/ManagedServices/ManagedServices/ManagedServices.csproj]: Why are you adding an empty xml element?

Comment (rajivnandivada) [reply src/ManagedServices/ManagedServices/ManagedServices.csproj]: That was unintended. I removed it.

**Comment by rajivnandivada** [comment]
> In order to go to 1.0, you need: (1) Service to be GA and have a stabe API; (2) Opt in to the breakign change policy [no breakign changes except during designated breakign change releases, and then, at least 2 months of previous notice]; (3) Provide us with your are path for ICM (you cna provide here: https://github.com/Azure/azure-powershell-pr/wiki/Azure-PowerShell-Partner-Contacts.

@markcowl Hey Mark, I have updated the IcM owners. We are also ok with opting into the breaking policy. Do I just need to add a documentation\upcoming-breaking-changes.md? Thank you.

File: src/ManagedServices/ManagedServices/Az.ManagedServices.psd1 [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.0.1'
+ModuleVersion = '1.0.0'
```
Comment (markcowl) [inline src/ManagedServices/ManagedServices/Az.ManagedServices.psd1]: Please revert this version change and update the changelog instead.  We will update the version in this case.

File: src/ManagedServices/ManagedServices/ChangeLog.md [hunk 3]
```diff
@@ -21,3 +21,6 @@
 
 ## Version 0.0.1
 * Initial version of the ManagedServices PowerShell interface
+
```
Comment (markcowl) [inline src/ManagedServices/ManagedServices/ChangeLog.md]: No, changelog changes are made under the 'Upcomign release' heading

**Comment by rajivnandivada** [comment]
@markcowl could you please take a look? Hopefully this will make the milestone:2019-07-30? Thank you.

**Pull request merged**